experimental/FTheoryTools/test/hypersurface_models.jl

```diff
@@ -2,6 +2,8 @@
 # 1: Hypersurface models over concrete bases
 #############################################################
 
+rand_seed = 1234
```

Comment:
And this is another key change - fixing one `rand_seed` in each test file and then passing this seed to the calls of `singular_loci`.

I tried to repeat this for all relevant test files and doc tests. Hopefully I found them all.

(Of course, not yet clear if this simple seed passes the tests...)

Reply:
Thank you. Just done.